File: backend/src/main/java/com/jayway/kdag/graphql/Person.java
package com.jayway.kdag.graphql;

import graphql.annotations.GraphQLField;

/**
 * Created by erik on 2015-11-29.
 */
public class Person {

    @GraphQLField
    public String id;
    @GraphQLField
    public String firstName;
    @GraphQLField
    public String lastName;

}

File: backend/src/main/java/com/jayway/kdag/graphql/Person.kt
package com.jayway.kdag.graphql

import graphql.annotations.GraphQLField

/**
 * Created by erik on 2015-11-29.
 */
class Person {

    @GraphQLField
    var id: String? = null
    @GraphQLField
    var firstName: String? = null
    @GraphQLField
    var lastName: String? = null

}
